jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/internal/ResponseHttpFields.java

```diff
@@ -203,4 +243,59 @@ public String toString()
     {
         return _fields.toString();
     }
+
+    /**
+     * A marker interface for {@link HttpField}s that cannot be {@link #remove(HttpHeader) removed} or {@link #clear() cleared}
+     * from a {@link ResponseHttpFields} instance.   Persistent fields are not immutable in the {@link ResponseHttpFields}
+     * and may be replaced with a different value.
+     */
```

Comment:
Does this mean persistent fields cannot be removed but can be overwritten? If this understanding is correct, the javadoc should maybe state that more explicitly.